guides/common/modules/con_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -0,0 +1,7 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="configuring-compliance-policy-deployment-methods_{context}"]
+= Configuring compliance policy deployment methods
+
+Use one the following procedures to configure {Project} for the method that you have selected to deploy compliance policies.
+You will select one of these methods when you later xref:common/modules/proc_creating-a-compliance-policy.adoc#Creating_a_Compliance_Policy_{context}[create a compliance policy].
```

Comment:
This sounds off to me: the first sentence states "use one", then second "you will".

Maybe reword to "Configure a method to deploy compliance policies to host. This is required to create a compliance policy." I am unsure if the second part is even necessary.

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214